analysis_options.yaml

```diff
@@ -0,0 +1,45 @@
+include: package:lint/analysis_options.yaml
+
+analyzer:
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
+  errors:
+    prefer_const_constructors: warning
+  exclude:
+    - "**/*.g.dart"
+    - "**/*.freezed.dart"
+    - "**/*.config.dart"
+    - "lib/gen/**"
+    - "test/**"
+
+linter:
+  rules:
+    ## MCND Custom
+    avoid_types_on_closure_parameters: true
+    sort_pub_dependencies: false
+
+    # ------ Disable individual rules ----- #
+    #                 ---                   #
+    # Turn off what you don't like.         #
+    # ------------------------------------- #
+
+    # Use parameter order as in json response
+    always_put_required_named_parameters_first: false
+
+    # Util classes are awesome!
+    avoid_classes_with_only_static_members: false
+
+
+    # ------ Enable individual rules ------ #
+    #                 ---                   #
+    # These rules here are good but too     #
+    # opinionated to enable them by default #
+    # ------------------------------------- #
+
+    # Make constructors the first thing in every class
+    sort_constructors_first: false
+
+    prefer_single_quotes: true
+    # public_member_api_docs: true
+    always_specify_types: false
```

Comment:
false??

Reply:
Enabling it will make the code much more complex and verbose
https://github.com/dart-lang/linter/issues/1620